tests/jquery.flot.Test.js

```diff
@@ -347,7 +347,7 @@ describe('flot', function() {
             [true, false].forEach(function(show) {
                 var series = {
                         lines: { show: show, zero: show },
-                        bars: { show: !show, zero: !show },
+                        bars: { show: !show, zero: !show, barWidth: 0.8 },
```

Comment:
If barWidth is not specified, will it not be the default 0.8? Is this really needed?

Reply:
The series is explicitly defined within the test, so the default values are never considered. This wasn't previously needed because a call to adjustSeriesDataRange, and later, computeBarWidth, did not take into account the defined barWidth value and instead used a hardcoded value of 0.8.